src/grids/MasonryGrid.ts

```diff
@@ -281,8 +267,8 @@ export interface MasonryGrid extends Properties<typeof MasonryGrid> {
 
 
 /**
- * The number of columns. If the number of columns is 0, it is automatically calculated according to the size of the container.
- * @ko 열의 개수. 열의 개수가 0이라면, 컨테이너의 사이즈에 의해 계산이 된다. (default: 0)
+ * The number of columns. If the number of columns is 0, it is automatically calculated according to the size of the container.  Can be used instead of outlineLength.
+ * @ko 열의 개수. 열의 개수가 0이라면, 컨테이너의 사이즈에 의해 계산이 된다. outlineLength 대신 사용할 수 있다. (default: 0)
```

Comment:
How about using `@default 0` in jsdoc instead?

Reply:
@WoodNeck 

Oh, very nice!!.